harvester_e2e_tests/apis/test_settings.py

```diff
@@ -162,3 +162,35 @@ def test_invalid_S3(self, api_client):
             f"S3 backup-target should check key/secret/bucket/region"
             f"API Status({code}): {data}"
         )
+
+
+@pytest.mark.p0
+@pytest.mark.negative
+@pytest.mark.settings
+@pytest.mark.settings
+@pytest.mark.skip_version_before('v1.3.1')
+class TestUpdateInvalidKubeconfigTTLMinuteSetting:
```

Comment:
Any particular reason to have respective classes?
I think we can refer `TestUpdateInvalidStorageNetwork` to have just one 
`TestUpdateKubeconfigDefaultToken`.

Reply:
That does make sense to condense them :smile: :+1:  - into one test class since there is not particularly a lot of negative or positive cases that require complex steps :smile: - I've gone ahead and implemented this in the most recent push.